config/locales/en.yml

```diff
@@ -1362,6 +1362,10 @@ en:
         work_package:
           is_not_a_valid_target_for_time_entries: "Work package #%{id} is not a valid target for reassigning the time entries."
           attributes:
+            id:
+              format: "%{message}"
+              lack_of_permission: "Cannot add child because you don't have permissions to edit the selected work package."
+              blank: "ID can't be blank."
```

Comment:
I understand that you tweaked format and blank so that the lack of permission error can be displayed without the "ID" prefix, and blank with the "ID" prefix. But I do not quite like having to do this: it may have an impact on how current and future errors are displayed on other places.

Wouldn't it be better to simply do `child.errors.add(:id, I18n.t(:errors.cannot_add_child_because_of_lack_of_permission))`?

Reply:
Actually, I'm wrong: the attribute prefix is always added.
Better keep it like it is.